q2_diversity/_beta/_method.py

```diff
@@ -63,6 +76,31 @@ def beta_phylogenetic(table: biom.Table, phylogeny: skbio.TreeNode,
     return results
 
 
+def beta_phylogenetic_alt(table: BIOMV210Format, phylogeny: NewickFormat,
+                          metric: str, n_jobs: int=1,
+                          variance_adjusted: bool=False,
+                          alpha=None,
+                          bypass_tips: bool=False) -> skbio.DistanceMatrix:
+
+    metrics = phylogenetic_metrics_alt_dict()
+
+    if metric not in metrics:
+        raise ValueError("Unknown metric: %s" % metric)
+
+    if alpha is not None and metric != 'generalized_unifrac':
+        raise ValueError('The alpha parameter is only allowed when the choice'
+                         ' of metric is generalized_unifrac')
```

Comment:
The "opposite" of this causes `unifrac` to raise an error, but the user is too far removed to be able to pinpoint the cause.

So we should add a branch to either set `alpha=1` when `alpha == None` or to explain that `alpha` needs to be set. Either should work.

Reply:
Great catch!!! 👏